packages/react-events/src/dom/Press.js

```diff
@@ -75,24 +75,24 @@ type PressEventType =
   | 'presschange';
 
 type PressEvent = {|
-  button: 'primary' | 'auxillary',
-  defaultPrevented: boolean,
-  target: Element | Document,
-  type: PressEventType,
-  pointerType: PointerType,
-  timeStamp: number,
+  altKey: boolean,
+  buttons: 1 | 4,
```

Comment:
Shouldn't this be `1 | 2 | 4` if its bitwise?

Reply:
We don't use right-click anymore in this responder

Reply:
Also looks like `pointerup` always have `buttons` as `0`, so will need that too.